ClassMethod/src/UniversityGroup.java

```diff
@@ -0,0 +1,51 @@
+import java.util.Arrays;
+
+public class UniversityGroup {
+    private final String groupName;
+    private final int startYear;
+    private int endYear;
+    private String[] studentArray;
+    public String getGroupName() {
+        return groupName;
+    }
+    public int getStartYear() {
+        return startYear;
+    }
+    public String[] getStudentArray() {
+        return studentArray;
+    }
+    public int getEndYear() {
+        return endYear;
+    }
+    public int setEndYear() {
+        return startYear + 5;
```

Comment:
This should be actually set of endYearField

Reply:
I removed it